src/IPBlock.php

```diff
@@ -53,7 +55,7 @@ abstract class IPBlock implements \ArrayAccess, \IteratorAggregate, \Countable
     protected $nb_addresses;
 
     /**
-     * @var string Either "IPv4" or "IPv6"
+     * @var string|IP
```

Comment:
This is always a string I think, never an IP object.

Reply:
Agreed.